app/views/shared/activities/_previous_records.html.erb

```diff
@@ -6,10 +6,11 @@
       </div>
       <div class="col-lg-11 col-sm-12 align-self-center">
         <%= label %> previously completed
+        <%= t('activity.previously_completed', label: label) %>
         <strong>
           <%= link_to "#{human_counts(list)}", school_timeline_path(school) %>
         </strong>
-        <%= list.count > 1 ? ' most recently on ' : ' on ' %>
+        <%= list.count > 1 ? " #{t('activity.completed_most_recently_on')} " : " #{t('activity.completed_on')} " %>
         <strong>
           <%= link_to "#{nice_dates(most_recent)}", path %>
```

Comment:
I've made a note to revisit this - may need a bit of refactoring to work with Welsh translations.